.circleci/config.yml

```diff
@@ -47,33 +47,22 @@ aliases:
       set -u
       set -o pipefail
 
-      # This Shallow Git Clone code is adapted from what the standard CircleCI `checkout` command does for the case of an external PR (link to example below):
-      # https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/49817/workflows/dc195ea6-ac06-4de1-9edf-4c949427b5fb/jobs/1430976/parallel-runs/0/steps/0-101
-      ### git clone --no-checkout "$CIRCLE_REPOSITORY_URL" .
-      ### git fetch --force origin +refs/pull/18748/head:refs/remotes/origin/pull/18748
-      ### git checkout --force -B "$CIRCLE_BRANCH" "$CIRCLE_SHA1"
-      ### git --no-pager log --no-color -n 1 --format='HEAD is now at %h %s'
-
       # Set up SSH access
       # This SSH key is the current github.com SSH key as of June 2023, but it will need to be changed whenever github changes their key (probably every few years)
       GITHUB_SSH_KEY="AAAAC3NzaC1lZDI1NTE5AAAAIOMqqnkVzrm0SdG6UOoqKLsabgH5C9okWi0dh2l9GKJl"
       mkdir -p ~/.ssh
       echo github.com ssh-ed25519 $GITHUB_SSH_KEY >> ~/.ssh/known_hosts
 
-      # Take a different clone path depending on if it's a tag, a PR from an external repo, or the normal case
+      git clone --depth 1 --no-checkout "$CIRCLE_REPOSITORY_URL" .
+      git fetch --depth 1 origin "$CIRCLE_SHA1"
+
       if [ -n "${CIRCLE_TAG-}" ]; then
```

Comment:
I wonder if the code for tags ever did anything. It seems that we are checking a variable called `CIRCLE_TAG-` here. I guess we can keep this code, but seems strange to me.

Reply:
The variable does exist for builds triggered from tags, we just don't have any like that.

It was originally added because it was part of the built-in checkout command, which @HowardBraham referred to when this shallow checkout was originally written. We kept it as similar as possible in behavior, except for the part where it was shallow (and thus much faster).